ph5/clients/ph5toms.py

```diff
@@ -1184,6 +1186,11 @@ def get_args():
         "[array].[seed_station].[seed_channel].[start_time]",
     )
 
+    parser.add_argument(
+        "--epoch", action="store_true", default=False,
+        help="Log the epoch of the miniseed files to the terminal"
```

Comment:
This is not important but I feel the argument name doesn't really describe what it does. I recommend "log_e"